src/client/Microsoft.Identity.Client/AppConfig/PublicClientApplicationBuilder.cs

```diff
@@ -239,7 +254,9 @@ public PublicClientApplicationBuilder WithParentActivityOrWindow(Func<IWin32Wind
 
             return WithParentFunc(() => (object)windowFunc());
         }
+#endif
 
+#if DESKTOP || NET5_WIN || NET_CORE
```

Comment:
Is this  only applicable on Windows desktop? Perhaps we should have a flag for that?

Reply:
This itnerface is part of WinForms, and we don't have a dependency on WinForms from NET_CORE